data/private

```diff
@@ -126,22 +126,24 @@ full:instant.arubanetworks.com
 full:setmeup.arubanetworks.com
 
 # ASUS router
-full:router.asus.com
+router.asus.com
+asusrouter.com
 
 # KDE
 networkcheck.kde.org
 
+# Netgear router
+routerlogin.com
```

Comment:
Use the `full:` prefix whenever you can to avoid mismatch.

Reply:
Using `full:` can't match `www.routerlogin.com`, so just use `domain:routerlogin.com` for all situations.

Reply:
> The url on the label of router is `routerlogin.com`, using `full:` can't match `www.routerlogin.com`, which is accessible too, so just use `domain:routerlogin.com` for all situations since we don't know which url user may input.

Then why not just let it be `full:www.routerlogin.com`

Reply:
Added `full:routerlogin.com` and `full:www.routerlogin.com`.